tool/evaluate-file.ts

```diff
@@ -40,16 +40,25 @@ export function initArgparse(subparsers : argparse.SubParser) {
         required: true,
         help: 'Path to ThingTalk file containing class definitions.'
     });
+    parser.add_argument('--parameter-datasets', {
+        required: true,
+        help: 'Path to parameter dataset manifest.'
+    });
     parser.add_argument('--predictions', {
         required: true,
         type: fs.createReadStream,
-        help: 'Prediction results (in TSV format: id, prediction)'
+        help: 'Prediction results (in TSV format: id, sentence, target, prediction)'
     });
```

Comment:
I think at some point we wanted this file format to be compatible with `genienlp predict`, which only does `id` and `prediction`. Not sure what happened to that.

Reply:
not sure, apparently that won't work here...